.github/workflows/nrm-ci.yml

```diff
@@ -0,0 +1,47 @@
+name: nrm-CI
+on: [push, pull_request]
+jobs:
+    nrm-ci:
+
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                python-version: [3.8]
+                check: [style, license, install]
+
+        env:
```

Comment:
Unless I'm mistaken, these are not necessary here. The workflow is not going to modify the PRs or anything else.

Reply:
`GITHUB_TOKEN` is probably unnecessary, good catch. It's been a bit (I'll need to check), but I think `TERM` changes Actions' output to have colors.